Maddori.Apple/Maddori.Apple/Screen/Home/Home/TeamManageViewController.swift

```diff
@@ -46,21 +44,23 @@ final class TeamManageViewController: BaseViewController {
     }
     
     override func render() {
+        
+
         view.addSubview(scrollView)
         scrollView.snp.makeConstraints {
             $0.edges.equalToSuperview()
         }
         
-        scrollView.addSubview(tempView)
-        tempView.snp.makeConstraints {
-            $0.width.equalTo(UIScreen.main.bounds.width)
-            $0.height.equalTo(275) // FIXME: 수치 바꿀것
+        scrollView.addSubview(chageTeamView)
+        chageTeamView.snp.makeConstraints {
             $0.top.equalToSuperview()
+            $0.width.equalToSuperview()
+            teamCount == 0 ? $0.height.equalTo(280) : $0.height.equalTo((teamCount * 59) + ((teamCount-1) * 8) + 80)
```

Comment:
p4;

```suggestion
            teamCount == 0 ? $0.height.equalTo(280) : $0.height.equalTo((teamCount * 59) + ((teamCount - 1) * 8) + 80)
```

앞 뒤로 띄어쓰기 부탁합니다!

Reply:
p3;

추가적으로 
```swift
    private enum Size {
        static let teamSectionHeight = 59
        static let teamSectionSpacing = 8
    }
```
이렇게 enum으로 Size에 대해 적어두고 사용하는건 어떨까요??
지금 당장은 저 숫자들의 의미를 파악하는데 큰 어려움은 없지만, 조금만 지나도 의미를 이해하기가 어려울 것 같아서요!
변수명은 입맛에 따라 수정하셔도 좋을 것 같습니다!

뒤에 80은 어떤 의미인지 모르겠어서 추가를 못했습니다!

```suggestion
            teamCount == 0 ? $0.height.equalTo(280) : $0.height.equalTo((teamCount * Size.teamSectionHeight) + ((teamCount - 1) * Size.teamSectionSpacing) + 80)
```

Reply:
[a664823](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/319/commits/a66482327d4def2db322bb8ea2057ee212e89dbd)

네네 맞습니다 감사합니다

커밋에 offset(28) 이 딸려갔는데 다음 커밋에서 지웠습니다.[a2b65e4](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/319/commits/a2b65e4d8f69bb5b8fe91a0aab7a8d105761c568)